### ISSUE TITLE
Color contrast not appropriate for fadded attributes in launch.json

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#56794]
**Environment Details:**
VSCode Version : 1.24.0
OS Version : Win10

**Additional Details:**
MAS Violated : MAS 1.4.3
Tools Used : Color Contrast Analyzer

**Repro Steps:**
1)Navigate to Settings-->Select color theme -->select light visual studio
2)Check color contrast ratio for the Editor content on the screen.

**Actual:**
Color contrast ratio is less than 4.5:1 for the black colored content when focused.
Similar issue is observed in following themes:
light Visual studio
Light+(Default light)
Quiet light
Solarized light
Monokai Dimmed

**Expected:**
Color contrast ratio should be more than 4.5:1 for the content on a screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

](url)
**User Impact:**
The low vision/visually impaired users will not be able to view the content properly.

**MAS Reference:**
MAS 1.4.3 [ https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

](url)

**Attachment For Reference:**
![22852_a11y_vscode_colortheme_colorcontrast](https://user-images.githubusercontent.com/39204909/44325893-acaecb00-a477-11e8-8c67-0cd4aef43393.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
@misolori can you maybe look into this?
Here's a [code pointer](https://github.com/Microsoft/vscode/blob/3d36db7dedfa195c6a4bb726d43a6de5a8c5b8eb/extensions/configuration-editing/src/extension.ts#L15) for the color 
The idea behind this is that these fields are not so important thus they are grayed out a bit. So making them much more visible would defeat the purpose.

**Comment by misolori** [comment]
The closest color that passes on white would be `#777`:
![image](https://user-images.githubusercontent.com/35271042/44413462-416d0380-a520-11e8-9deb-1c39bc99810f.png)

**Comment by AccessibilityTestingTeam-TCS** [comment]
Checked and verified in below version i.e 1.27.0-Insiders.

The issue reproduced in following themes:

1)Light visual studio
2)Light+(Default light)
3)Quiet light 
4)Solarized light

**Comment by isidorn** [comment]
@misolori thanks for finding the right color
@AccessibilityTestingTeam-TCS thanks for checking with latest, please only test with that version

**Comment by roblourens** [comment]
Solarized Light:

![image](https://user-images.githubusercontent.com/323878/44872933-a7a60480-ac4b-11e8-9ed3-f8533cad3615.png)

Quiet light:

![image](https://user-images.githubusercontent.com/323878/44872952-b987a780-ac4b-11e8-88e8-862e52dbf7c4.png)

Dark:

![image](https://user-images.githubusercontent.com/323878/44873023-f2c01780-ac4b-11e8-8145-6f92c8da7d6b.png)

**Comment by misolori** [comment]
FYI: We're only fixing for themes we own (Dark, Light, HC).

@isidorn any way to set this color for each dark + light theme? There isn't a color that's going to work in both themes. For light, `#757575` passes and for dark `#878787` passes as well.

Light
<img width="50%" src="https://user-images.githubusercontent.com/35271042/44875361-cb207d80-ac52-11e8-922d-9fb3c2f5d135.png">

Dark
<img width="50%" src="https://user-images.githubusercontent.com/35271042/44875318-aa582800-ac52-11e8-9208-6436a342c6ec.png">

**Comment by isidorn** [comment]
We can do it per theme - I think. I will use the two colors you provided.
We can definetely use one of the existing colors from here https://code.visualstudio.com/docs/getstarted/theme-color-reference

And providing custom colors per theme from an extension - I have to experiment, should be possible.
